src/main/resources/application.yaml

```diff
@@ -55,6 +55,10 @@ idam:
     id: 'bsp'
     secret: ${IDAM_CLIENT_SECRET}
     redirect_uri: ${IDAM_CLIENT_REDIRECT_URI}
+  users:
+    bulkscan:
+      username: ${IDAM_USERS_BULKSCAN_USERNAME:notset}
+      password: ${IDAM_USERS_BULKSCAN_PASSWORD:notset}
```

Comment:
please explain

Reply:
I think this whole element should go away - users are retrieved from environment variables

Reply:
Have removed